.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Build docker image and publish
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Login to Docker hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_HUB_USERNAME }}
+        password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+    - name: Set up Docker Build
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+    
+    - name: Build and push
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        context: ./simplq
+        file: .Dockerfile
+        push: true
+        tags: ${{ secrets.DOCKER_HUB_USERNAME }}/simpliQ-frontend:latest
```

Comment:
Created [daltonfury42/simplq-frontend ](https://hub.docker.com/repository/docker/daltonfury42/simplq-frontend), and added DOCKER_HUB_USERNAME and DOCKER_HUB_ACCESS_TOKEN to org secrets.

Can you rename to the repo name